Add note chatbot not working in current excersize (#210)
sveltejs/learn.svelte.dev: content/tutorial/01-svelte/07-lifecycle/03-update/README.md

title: beforeUpdate and afterUpdate
---

> > The Eliza Chatbot in this exercise don't currently work. You can switch to the old tutorial instead: https://svelte.dev/tutorial/update

Reviewer: ```suggestion
> The Eliza Chatbot in this exercise doesn't currently work. For now, you can use the old tutorial instead: https://svelte.dev/tutorial/update
```
